_search-plugins/search-relevance/index.md

```diff
@@ -32,7 +32,7 @@ Alternatively, you can add sample data in OpenSearch Dashboards using the follow
 1. Select **Add sample data**.  
 1. Choose one of the built-in datasets and select **Add data**.
 
-## Using search relevance in OpenSearch Dashboards
+## Using Compare Search Results in OpenSearch Dashboards
```

Comment:
Is "Compare Search Results" a UI element? If not, we shouldn't capitalize it.

```suggestion
## Using compare search results in OpenSearch Dashboards
```

Reply:
Compare Search Results is the name of the feature, and the name of the UI page. As per editorial review of the previous PR, we are capitalizing this feature. @natebower please confirm. Thank you!